graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/BaseDgsQueryExecutor.kt

```diff
@@ -89,15 +90,13 @@ object BaseDgsQueryExecutor {
         val graphQLBuilder =
             GraphQL.newGraphQL(graphQLSchema)
                 .preparsedDocumentProvider(preparsedDocumentProvider)
-                .instrumentation(chainedInstrumentation)
                 .queryExecutionStrategy(queryExecutionStrategy)
                 .mutationExecutionStrategy(mutationExecutionStrategy)
-                .subscriptionExecutionStrategy(SubscriptionExecutionStrategy())
```

Comment:
`SubscriptionExecutionStrategy(this.defaultExceptionHandler)` is what gets aded by default if no execution strategy is supplied for subscriptions; it seems better to rely on the builder's default, imo, unless we were doing this to avoid having the defaultExceptionHandler set?